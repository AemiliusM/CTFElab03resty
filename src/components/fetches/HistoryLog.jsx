import React from "react";

export default function HistoryLog({ method, url }) {
    return (
        <article>
            <p>{method}</p>
            <span>{url}</span>
        </article>
    )
}