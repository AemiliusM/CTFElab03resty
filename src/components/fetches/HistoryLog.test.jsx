import React, { Component } from 'react';
import { render } from '@testing-library/react';
import HistoryLog from './HistoryLog';

describe('history test', () => {
    it('should render a single history', () => {
        const { asFragment } = render(
            <HistoryLog
                method='GET'
                url='https://jsonplaceholder.typicode.com/todos/1'
                />
        );
        expect(asFragment()).toMatchSnapshot();
    });
})