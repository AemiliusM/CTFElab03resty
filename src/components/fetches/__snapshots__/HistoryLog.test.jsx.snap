// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`history test should render a single history 1`] = `
<DocumentFragment>
  <article>
    <p>
      GET
    </p>
    <span>
      https://jsonplaceholder.typicode.com/todos/1
    </span>
  </article>
</DocumentFragment>
`;
